src/Database/Schema/CachedCollection.php

```diff
@@ -70,7 +70,7 @@ public function listTables(): array
     /**
      * {@inheritDoc}
      */
-    public function describe(string $name, array $options = []): TableSchemaInterface
+    public function describe(string $name, array $options = []): TableSchema
```

Comment:
This might cause problems with DebugKit as it has an implementation of Schema\Collection as well.